src/MoBi.Engine/EngineRegister.cs

```diff
@@ -11,15 +12,34 @@ public override void RegisterInContainer(IContainer container)
          container.AddScanner(x =>
          {
             x.AssemblyContainingType<EngineRegister>();
+            x.WithConvention(new OSPSuiteRegistrationConvention(registerConcreteType: true));
             x.ExcludeType<UnitDefinitionImporter>();
+            x.ExcludeType<CompartmentImporter>();
+            x.ExcludeType<SpeciesImporter>();
+            x.ExcludeType<ParameterImporter>();
             x.ExcludeType<FunctionDefinitionImporter>();
-            x.WithConvention(new OSPSuiteRegistrationConvention(registerConcreteType: true));
+            x.ExcludeType<ReactionImporter>();
+            x.ExcludeType<AssignmentImporter>();
```

Comment:
We do not need this. See current change in develop where we register by convention

Reply:
Right, only ASTHandler was missing.